assembly/std/op-datastore/index.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * This namespace is used to interact with the **operation datastore**.
+ * It is located at **std/op-datastore**.
+ *
+ * @remarks
```

Comment:
I guess it's a private remarks as it doesn't change the way the dev interact with theis library

Reply:
The reason why I did this listing is for the exemple of the namespace 'Utilities'  which has many different sub-folders to categorize functions, so the purpose was to explain to the user theses implicit sub-namespaces, but maybe we should do theses explicitly to avoid such redundant function listing and have proper descriptions ?

Reply:
you're right.